sonic-psud/tests/test_PsuChassisInfo.py

```diff
@@ -189,17 +197,17 @@ def test_consumed_power(self):
 
     def test_power_budget(self):
         chassis = MockChassis()
-        psu = MockPsu(True, True, "PSU 1", 0)
+        psu1 = MockPsu("PSU 1", 0, True, True)
         psu1_power = 510.0
         psu.set_maximum_supplied_power(psu1_power)
```

Comment:
Line #194, object is psu1..so, should be psu1.set_maximum_supplied_power()? Please check other instances like line #223

Reply:
Good catch, thanks! Fixed in latest commit.